src/lib389/lib389/instance/options.py

```diff
@@ -114,6 +115,8 @@ def parse_inf_config(self, config):
                     v = config.getint(self._section, k)
                 elif self._type[k] == bool:
                     v = config.getboolean(self._section, k)
+                elif self._type[k] == float:
```

Comment:
This should be an int not a float. But better would be a way to parse the units.

Reply:
Fixed (it is now a string)